static/js/app.js

```diff
@@ -39,6 +39,26 @@ $(function(){
           });
     });
 
+    $('#validate').on('click', function(event){
+        event.preventDefault();
+        var postData = $('#post-data').val();
+        $.post('/validate', $('#post-data').val())
+          .done(function(data){
+            $(".alert").removeClass('alert-success alert-danger hidden');
+            if (data.indexOf("true") > 0) {
+              $(".alert").addClass('alert-success').text("Validation result: " + data.substr(0, 100) + "...");
```

Comment:
Don't think you need the ' .substr(0, 100) + "..." ' part in each of these cases, this was in the original code as a means to abbreviate what the post data message.